console-subscriber/tests/spawn.rs

```diff
@@ -10,6 +10,7 @@ use support::{assert_tasks, spawn_named, ExpectedTask};
 /// spawned the child task). In this scenario, that would result in the parent
 /// having 3 polls counted, when it should really be 1.
 #[test]
+#[allow(clippy::async_yields_async)]
```

Comment:
in general, it's nice if allow attributes for lints can have a comment explaining why they're allowed in this instance. it would be nice if you're able to add one here, although it's not a huge deal.

Reply:
Good suggestion! Added.